upb/bindings/lua/upb.c

```diff
@@ -1056,6 +1064,128 @@ static const struct luaL_Reg lupb_enumdef_m[] = {
 };
 
 
+/* lupb_filedef ***************************************************************/
+
+void lupb_filedef_pushwrapper(lua_State *L, const upb_filedef *f,
+                              const void *ref_donor) {
+  lupb_refcounted_pushwrapper(L, upb_filedef_upcast(f), LUPB_FILEDEF, ref_donor,
+                              sizeof(void *));
+}
+
+void lupb_filedef_pushnewrapper(lua_State *L, const upb_filedef *f,
```

Comment:
I think I was trying to increase readability by removing the repeated letter, a la getty: https://en.wikipedia.org/wiki/Getty_(Unix)

How does it strike you as a reader?